man/po/da.po

```diff
@@ -5471,7 +5471,7 @@ msgstr ""
 
 #: login.defs.5.xml:34(para) login.1.xml:34(para)
 msgid "Enable logging of successful logins."
-msgstr "Aktiver logning af succesfulde logind."
+msgstr "Aktiver logning af successfulde logind."
```

Comment:
Was this change intentional? I'm not a Danish speaker so I'm not sure, but it seems like you replaced all `succesful` instances by `successful`

Reply:
Looking at translate.google.com, it looks like succesfulde is 'success' while successfulde is successful.  So I'll take it these are intentional, and merge.

Thanks, @uniontech-lilinjie , and thanks for looking, @ikerexxe